sklearn/compose/_column_transformer.py

```diff
@@ -670,16 +672,16 @@ def make_column_transformer(*transformers, **kwargs):
     Parameters
     ----------
     *transformers : tuples
-        Tuples of the form (transformer, column(s)) specifying the
+        Tuples of the form (transformer, columns) specifying the
         transformer objects to be applied to subsets of the data.
 
-        transformer : estimator or {'passthrough', 'drop'}
+        transformer : {'drop', 'passthrough'} or estimator
             Estimator must support :term:`fit` and :term:`transform`.
             Special-cased strings 'drop' and 'passthrough' are accepted as
             well, to indicate to drop the columns or to pass them through
             untransformed, respectively.
-        column(s) : string or int, array-like of string or int, slice, \
-boolean mask array or callable
+        columns : str,  array-like of str, int, array-like of int, slice, \
+            array-like of bool or callable
```

Comment:
let's use the same doc as for the class:

```
        columns : str, array-like of str, int, array-like of int, \
                array-like of bool, slice or callable
```

Reply:
Yes, good catch. Corrected.